common/app/templates/inlineJS/blocking/polyfills/details.scala.js

```diff
@@ -0,0 +1,49 @@
+@()
+
+(function (window, document) {
+    var weShouldPolyfill = !('open' in document.createElement('details'));
+
+    function toggleDetailsState(details) {
+        if (details.hasAttribute('open')) {
+            details.removeAttribute('open');
+        } else {
+            details.setAttribute('open', 'open');
+        }
+    }
+
+    function handleEvent(event) {
+        var targetNode = event.target;
+
+        while (targetNode.nodeName.toLowerCase() !== 'summary' && targetNode !== document) {
+            targetNode = targetNode.parentNode;
+        }
+        if (targetNode.nodeName.toLowerCase() === 'summary') {
+            toggleDetailsState(targetNode.parentNode);
+        }
+    }
+
+    function bindEvents() {
+        document.addEventListener('click', handleEvent);
```

Comment:
Yeah, that's a good shout. It would stop people accidentally breaking the `<details>` functionality if they use `stopPropagation`